src/components/Pools/Remove/index.tsx

```diff
@@ -36,15 +36,33 @@ const Remove = ({ stakingInfo, version, onClose, redirectToCompound }: WithdrawP
 
   const { userLiquidity } = useDerivedBurnInfo(currencyA ?? undefined, currencyB ?? undefined);
 
+  function onComplete(percetage: number, type: REMOVE_TYPE) {
+    if (percetage >= 100) {
+      setShowRemoveTab(false);
+      // if remove all from farm it will show the liquidity tab
+      if (type === REMOVE_TYPE.FARM) {
+        setRemoveType(REMOVE_TYPE.LIQUIDITY);
+      } else {
+        // if remove all liquidity and have a stake in farm it show farm tab
+        if (stakingInfo.stakedAmount && stakingInfo.stakedAmount.greaterThan('0')) {
+          setRemoveType(REMOVE_TYPE.FARM);
+        }
+      }
+    }
+  }
```

Comment:
there is some issues after this change in remove liquidity. please also test remove farm.

https://user-images.githubusercontent.com/6604146/224400376-f6fd88b3-5f30-45ff-adf3-1b73f334366d.mov